src/tests/functional/plugin/conformance/subgraphs_dumper/src/utils/model.cpp

```diff
@@ -52,7 +52,8 @@ find_models(const std::vector<std::string> &dirs, const std::string& regexp) {
                 } else {
                     continue;
                 }
-            } catch (std::exception& e) {
+            } catch (...) {
```

Comment:
```suggestion
            } catch (std::exception&) {
```
Will it it work also?